Manual.md

```diff
@@ -356,6 +356,8 @@ in this directory such as `${XBPS_BUILDDIR}/${wrksrc}`.
 
 - `XBPS_CROSS_TRIPLET` The target machine GNU type triplet, only available when cross-compiling.
 
+- `XBPS_CROSS_BASE` Full path to where dependencies for cross compilation are installed.
```

Comment:
Somehow this doesn't really explain where it actually points to, maybe an example (`/usr/${XBPS_CROSS_TRIPLET}`) would make that clearer